src/modal/modal-window.ts

```diff
@@ -40,12 +42,16 @@ export class NgbModalWindow implements OnInit,
   @Input() size: string;
   @Input() windowClass: string;
 
+  @Input() ariaLabelledBy: string;
```

Comment:
Could you please put in in the alphabetical order?

https://github.com/ng-bootstrap/ng-bootstrap/wiki/Contributions%3A-Code-conventions#component-internal-organization

Reply:
I missed that, my bad, I'll change it